chanzuckerberg/single-cell-data-portal
# chore: build FE container at build time

#3830

## Changes

- Builds the FE container at build time and not at runtime (like currently)
- Lower the memory requirement to 4G (minimum for the FARGATE configuration we have)
- Revert the healthcheck interval to 15s

## Testing steps
- https://ebezzi-fe-build-frontend.rdev.single-cell.czi.technology/
- Test the local application (it should work with `make local-sync`)
- Check that it works in dev and the container boots in a few seconds (through Cloudwatch)
- Check that the sitemaps are working (they're affected by this change)
